spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
+    before :each do
+      Division.delete_all
 
+      @division = Division.create(id: 5, name: "hey jude", date: Date.new(2016,12,25),
```

Comment:
Could you use let with the !, `let!`? [It runs the statement before each example](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let):

```
let!(:division1) { create(:division, date: Time.new(2016,12,25) }
let!(:division2) { create(:division, date: Time.new(2016,06,01) }
```

That also uses the factories so you only have to assign the data we care about for this test, and the factory sets the other stuff for a valid Division.